Cargo.toml

```diff
@@ -65,3 +65,6 @@ packed_simd = { version = "0.3.4", package = "packed_simd_2", features = ["into_
 [features]
 default = ["alloc"]
 alloc = ["zeroize/alloc"]
+
+[profile.dev]
+opt-level = 2
```

Comment:
This makes the tests run 5 times faster for me with `cargo test`, with compile times effectively the same